bosh-director/lib/bosh/director/deployment_plan/planner_factory.rb

```diff
@@ -229,16 +229,16 @@ def update_mapped_properties(mapped_properties, property_path, value)
           return mapped_properties
         end
 
-        def parse_numerical_arguments arg
+        def parse_arguments arg
```

Comment:
It looks like this method is public. Are there any existing unit tests for it? This functionality change looks like it would be good to have a unit test. This method also has two possible return types, string in the "%" case and int in the digit case. Can we refactor this to only have one return type and make sure that consumers of this method always properly handle strings as output?